documentation/docs/05-misc/03-typescript.md

```diff
@@ -177,9 +177,11 @@ You cannot use TypeScript in your template's markup. For example, the following
 	let count = 10;
 </script>
 
-<h1>Count as string: {count as string}!</h1> <!-- ❌ Does not work -->
+<!-- ❌ Does not work -->
+<h1>Count as string: {count as string}!</h1>
 {#if count > 4}
-	{@const countString: string = count} <!-- ❌ Does not work -->
+	<!-- ❌ Does not work -->
+	{@const countString: string = count}
```

Comment:
Can we revert the doc changes? We're going to rewrite most of these anyway, so no point in doing this now and risk merge conflicts in case we want to port some V4 changes to the main branch.

Reply:
I also want to revert this but I can not pass the lint ci.
Is it better to add this file to .prettierignore?

Reply:
Once I tried to update it.
https://github.com/sveltejs/svelte/pull/9609/commits/cba043c78677f9f336f3edfb4d0ad99db7140c08

Reply:
I'll look into on Monday more closely. If it's only related to prettier then we might need to add it. Although I don't understand how this wasn't failing before then

Reply:
I think because Svelte compiler without this version ca not parse the code due to type in @const tag.